security/encrypt-cluster-pod-traffic.md

```diff
@@ -81,80 +81,54 @@ kubectl -n calico-system set env daemonset/calico-node --containers="calico-node
 %>
 <label:OpenShift>
 <%
-To install WireGuard for OpenShift v4.6:
+To install WireGuard for OpenShift v4.8 (and above):
```

Comment:
I would suggest we only specify OCP 4.8, as newer versions will require a testing cycle.

Reply:
ok gotcha